packages/storybook/src/documentation/design-tokens/design-tokens.md

```diff
@@ -0,0 +1,15 @@
+# Design Tokens
+
+[digid]: https://digid.nl/
+[lagen]: /docs/design-tokens-lagen--docs
+[mijn-overheid]: https://www.mijnoverheid.nl/
+[nldoc]: https://www.digitoegankelijk.nl/nieuws/publicatietool-nldoc
+[rijkshuisstijl]: https://www.rijkshuisstijl.nl/publiek
+
+<!-- TODO: (LUX-27) linken "installeren" (einde vd paragraaf) naar installatiehandleiding -->
+
+De basis van het LUX design system is design tokens. Ze bepalen hoe je applicatie of website eruitziet. Ze leggen de designkeuzes vast in code. Om gebruik te maken van de LUX design tokens moet je deze eerst installeren.
+
+Bij Logius werken wij aan verschillende applicaties en websites. Denk aan [Digid][digid], [Mijn Overheid][mijn-overheid] en [NLDoc][nldoc]. Deze hebben allemaal hun eigen huisstijl, gebaseerd op de [Rijkshuisstijl][rijkshuisstijl]. De gebruikersinterface van deze applicaties worden gebouwd met LUX. Ze gebruiken de zelfde LUX componenten, maar die zien er wel allemaal anders uit. Dat doen wij met behulp van design tokens.
+
+In essentie zijn design tokens CSS variabelen die door LUX componenten worden toegepast. Om de verschillende huisstijlen waar te maken delen we de design tokens op in [verschillende lagen][lagen]. Daarmee kunnen wij makkelijk onderscheid maken tussen de verschillende huisstijlen, thema's, en modes zonder de componenten voor elke applicatie opnieuw te bouwen.
```

Comment:
"modes" worden in de lagen niet verder benoemd

Reply:
Klopt. Ik ga er eigenlijk van uit dat men dat begrijpt. Net als thema of huisstijl

Reply:
Er wordt wel genoemd dat men van `light` naar `dark` kan wisselen: https://github.com/nl-design-system/lux/pull/76/files#diff-3b0aa420ecfce4cb3b02e5a8a945e316559fdf5a352f0cc5a18ddc1dab979eadR17